### PR TITLE
Css layout classes

### PR DESCRIPTION
**Pull request by devtanole**
Screenshot 2024-08-12 at 4 06 00 PM

still having trouble with columns and identifying where to specify those classes. will edit.

### CODE REVIEW

File: css-layout-classes/css-layout-classes-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- Why do two div elements "vertically stack" on one another by default?
+  A: they are block display.
+
+- What is the default `flex-direction` of an element with `display: flex`?
+  A: row
+- What are the three primary components of a page layout? (Which helper classes do you need?)
+  A: footer, header and body
```
Comment (hesam) [inline css-layout-classes/css-layout-classes-quiz-notes.md:13]: They're looking for: Container, Row, Column

File: css-layout-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            <p>
+              Super Smash Bros. Ultimate received universal acclaim from both
+              critics and players, with some critics calling it the best
+              installment in the series. It received praise for its large amount
+              of content and fine-tuning of existing Smash gameplay elements,
+              although its online mode was widely criticized.
+            </p>
+          </div>
+          <div>
+            <img src=".//images/yoshi.png" />
```
Comment (hesam) [inline css-layout-classes/index.html:29]: Single slash '/' is correct. Double '//' is incorrec.t

**Review by hesam — CHANGES_REQUESTED** [review]
- Nice work! 
- Because you're not constraining the width of items in each row (flexbox), the width of rows don't end up equal. See if you could assign appropriate width setting classes for items (e.g. 'one-third-width' etc.) so that the sum of the widths always end up equal the width of the container and equal to other rows...
- Please see a few comments in the code/quiz.

File: css-layout-classes/index.html (hunk 2 above)
Comment (devtanole) [reply css-layout-classes/index.html:29]: Screenshot 2024-08-13 at 8 35 00 PM

have fixed my errors for quiz and my code with new attempt2.html and styles2.css

**Review by hesam — APPROVED** [review]
Looking good 👍

**Pull request merged**